### PR TITLE
Update the tutorials for Python API to fix consistent duplicate point errors.

### PR DESCRIPTION
**Pull request by atomoclast**
This PR is in conjunction with this current PR:

https://github.com/ros-planning/moveit/pull/737

I have updated the tutorials because of a consistent issue I was facing when following it with the Python API. 

When you attempt to plan a cartesian path following the Python API, you will often receive an error like this:  

`[ERROR] [1515384240.163191133, 270.462000000]: Trajectory message contains waypoints that are not strictly increasing in time.` 

Further investigation has shown that the cause of the issue is that in the tutorial, you append the starting point of the end effector into the `waypoints[]` list, so when the planner tries to plan to that waypoint, it sees that it will reach that position in 0 time and append that to the resultant path. This non-monotonic increase in the time causes the controller to throw this error because the waypoints are not unique in the timestamp.

I have added a warning in the planner in the above PR, and updated the tutorial to allow for better educational experiences and knowledge transfer to new/returning users of the Python API.

### CODE REVIEW

File: doc/pr2_tutorials/kinematics/src/kinematic_model_tutorial.cpp [hunk 1]
```diff
@@ -43,7 +43,7 @@
 
 int main(int argc, char **argv)
 {
-  ros::init(argc, argv, "right_arm_kinematics");
+  ros::init(argc, argv, "UR10_arm_kinematics");
```
Comment (davetcoleman) [inline doc/pr2_tutorials/kinematics/src/kinematic_model_tutorial.cpp]: this should not be changed

File: doc/pr2_tutorials/kinematics/src/kinematic_model_tutorial.cpp [hunk 2]
```diff
@@ -70,16 +70,10 @@ int main(int argc, char **argv)
   robot_model::RobotModelPtr kinematic_model = robot_model_loader.getModel();
   ROS_INFO("Model frame: %s", kinematic_model->getModelFrame().c_str());
 
-  // Using the :moveit_core:`RobotModel`, we can construct a
-  // :moveit_core:`RobotState` that maintains the configuration
-  // of the robot. We will set all joints in the state to their
-  // default values. We can then get a
-  // :moveit_core:`JointModelGroup`, which represents the robot
-  // model for a particular group, e.g. the "right_arm" of the PR2
-  // robot.
```
Comment (davetcoleman) [inline doc/pr2_tutorials/kinematics/src/kinematic_model_tutorial.cpp]: why remove this tutorial explanation?

File: doc/pr2_tutorials/kinematics/src/kinematic_model_tutorial.cpp [hunk 3]
```diff
@@ -114,7 +108,7 @@ int main(int argc, char **argv)
   // "r_wrist_roll_link" which is the most distal link in the
   // "right_arm" of the robot.
   kinematic_state->setToRandomPositions(joint_model_group);
-  const Eigen::Affine3d &end_effector_state = kinematic_state->getGlobalLinkTransform("r_wrist_roll_link");
+  const Eigen::Affine3d &end_effector_state = kinematic_state->getGlobalLinkTransform("ee_link");
```
Comment (davetcoleman) [inline doc/pr2_tutorials/kinematics/src/kinematic_model_tutorial.cpp]: revert

File: doc/pr2_tutorials/kinematics/src/kinematic_model_tutorial.cpp [hunk 4]
```diff
@@ -70,16 +70,10 @@ int main(int argc, char **argv)
...
-  // of the robot. We will set all joints in the state to their
-  // default values. We can then get a
-  // :moveit_core:`JointModelGroup`, which represents the robot
-  // model for a particular group, e.g. the "right_arm" of the PR2
-  // robot.
+
   robot_state::RobotStatePtr kinematic_state(new robot_state::RobotState(kinematic_model));
   kinematic_state->setToDefaultValues();
-  const robot_state::JointModelGroup *joint_model_group = kinematic_model->getJointModelGroup("right_arm");
+  const robot_state::JointModelGroup *joint_model_group = kinematic_model->getJointModelGroup("manipulator");
```
Comment (davetcoleman) [inline doc/pr2_tutorials/kinematics/src/kinematic_model_tutorial.cpp]: revert

File: doc/pr2_tutorials/kinematics/src/kinematic_model_tutorial.cpp (hunk 3 above)
Comment (atomoclast) [reply doc/pr2_tutorials/kinematics/src/kinematic_model_tutorial.cpp]: I need to check my branch again. I had a branch for tutorials on a blog post, which I think I forgot to reset to be level with your kinetic-devel before I made some changes. Let me fix this!

**Comment by atomoclast** [comment]
Let me fix this. It pulled in some changes for a different project I was working on.

**Comment by davetcoleman** [comment]
ping would still appreciate this fix!

**Comment by atomoclast** [comment]
@davetcoleman 

I actually closed this PR, because I pulled in an incorrect commit for a different project. This particular PR was redone and merged in here:
https://github.com/ros-planning/moveit_tutorials/pull/148

**Pull request closed without merging**